src/main/resources/view/DarkTheme.css

```diff
@@ -274,47 +311,56 @@
  */
 .button {
     -fx-padding: 5 22 5 22;
-    -fx-border-color: #e2e2e2;
+    -fx-border-color: white;
     -fx-border-width: 2;
-    -fx-background-radius: 0;
-    -fx-background-color: #1d1d1d;
+    -fx-border-radius: 0pt;
+    -fx-background-radius: 0pt;
+    -fx-background-color: black;
     -fx-font-family: "Segoe UI", Helvetica, Arial, sans-serif;
     -fx-font-size: 11pt;
     -fx-text-fill: #d8d8d8;
     -fx-background-insets: 0 0 0 0, 0, 1, 2;
 }
 
 .button:hover {
-    -fx-background-color: #3a3a3a;
+    -fx-background-color: #1fc7b8;
+    -fx-border-color: black;
+    -fx-text-fill: black;
 }
 
 .button:pressed, .button:default:hover:pressed {
-  -fx-background-color: white;
-  -fx-text-fill: #1d1d1d;
+    -fx-background-color: white;
+    -fx-text-fill: black;
+    -fx-border-color: black;
 }
 
-.button:focused {
-    -fx-border-color: white, white;
-    -fx-border-width: 1, 1;
-    -fx-border-style: solid, segments(1, 1);
-    -fx-border-radius: 0, 0;
-    -fx-border-insets: 1 1 1 1, 0;
-}
+/*.button:focused {*/
+/*    -fx-border-color: white;*/
+/*    -fx-border-width: 2;*/
+/*    -fx-border-radius: 5pt;*/
+/*    -fx-background-radius: 5pt;*/
+/*}*/
 
 .button:disabled, .button:default:disabled {
     -fx-opacity: 0.4;
-    -fx-background-color: #1d1d1d;
+    -fx-background-color: black;
     -fx-text-fill: white;
+    -fx-border-radius: 5pt;
+    -fx-background-radius: 5pt;
 }
 
-.button:default {
-    -fx-background-color: -fx-focus-color;
-    -fx-text-fill: #ffffff;
-}
+/*.button:default {*/
```

Comment:
Perhaps the comments can be removed?

Reply:
Good spot! Forgot to uncomment after editing. Will make the necessary changes!